data/wanderer ships.txt

```diff
@@ -728,3 +728,59 @@ ship "Deep River Transport"
 	explode "huge explosion" 20
 	"final explode" "final explosion large"
 	description "This is a Deep River freighter that has been converted into a transport ship for refugees."
+
+
+
+ship "Riptide"
+	sprite "ship/riptide"
+	attributes
+		category "Transport
+		licenses
+			Wanderer
+		"cost" 21120000
+		"shields" 38600
+		"hull" 22350
+		"required crew" 47
+		"bunks" 233
+		"mass" 350
+		"drag" 5.1
```

Comment:
I did a little bit of fine-tuning, and this should be raised to 5.8; I want to keep the max speed of the Riptide below that of the Autumn and Summer Leaf. This change means that a default Riptide will be only slightly faster than a default Autumn or Summer Leaf, but a refitted Leaf would be faster than the Riptide.